qemu/tests/cfg/verify_panic_status_with_pvpanic.cfg

```diff
@@ -0,0 +1,16 @@
+- verify_panic_status_with_pvpanic:
+    type = verify_panic_status_with_pvpanic
+    virt_test_type = qemu
+    monitors = 'qmp1'
+    monitor_type_qmp1 = qmp
+    enable_pvpanic = no
+    stop_kdump_command = 'systemctl stop kdump'
+    trigger_crash = 'echo c > /proc/sysrq-trigger'
+    qmp_command = "query-status"
+    qmp_check_info = "guest-panicked"
+    x86_64:
+        extra_params += " -device pvpanic,id=pvpanic0,ioport=0x0505 -no-shutdown"
```

Comment:
Why add this? I suppose with `enable_pvpanic = yes` it should has pvpanic by default.

Reply:
Ah, you wanna get the id of pvpanic device, you might get it from `vm.devices`, like:
```
ioport_param = {'ioport': params.get["ioport_pvpanic"]}
dev = vm.devices.get_by_params(ioport_param)
id = dev.get_param('id')
```
I understand it would be more complicated, but better not to use extra_params when you have another way, since this way the device will not be managed by vm.devices.

Reply:
Thanks for your guidance. In fact, we can just modify `check_info = "dev: pvpanic, id "pvpanic0""` to ` check_info= "dev: papanic"`,it is also works.